frontend/src/pages/Player/components/NoActiveSessionCard/NoActiveSessionCard.tsx

```diff
@@ -1,11 +1,11 @@
 import { Box, Callout, Text } from '@highlight-run/ui/components'
 import { useEffect } from 'react'
 
-import { useErrorPageNavigation } from '@/pages/ErrorsV2/ErrorsV2'
+import usePlayerConfiguration from '@/pages/Player/PlayerHook/utils/usePlayerConfiguration'
 import { useParams } from '@/util/react-router/useParams'
 
 const NoActiveSessionCard = () => {
-	const { setShowLeftPanel } = useErrorPageNavigation()
+	const { setShowLeftPanel } = usePlayerConfiguration()
```

Comment:
Think it makes sense to keep this bit of state separate for errors and sessions? Maybe not important, but thinking this is a behavior change for some people. Could possibly move to a reactive var rather than relying on a context 🤔